jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildConfiguration.java

```diff
@@ -391,14 +296,28 @@ public RegistryCredentials getKnownTargetRegistryCredentials() {
     return knownTargetRegistryCredentials;
   }
 
+  public Instant getCreationTime() {
```

Comment:
Hence, down here, these could all be just a `getContainerConfiguration`.

Reply:
Are we expecting the user to use these getters?

Reply:
Wait nevermind

Reply:
This will require changing code in a bunch of other files, so I'll leave this for another PR.